_includes/breadcrumbs.html

```diff
@@ -5,32 +5,36 @@
 <div id="breadcrumbs_wrapper">
 {% if page.categories.size > 1 %}
     <div id="breadcrumbs">
-        <ul>
+        <ul itemscope="" itemtype="http://schema.org/BreadcrumbList">
```

Comment:
nice! So it will be visible in Google, if anyone uses it?
https://developers.google.com/search/docs/advanced/structured-data/breadcrumb

Reply:
>So it will be visible in Google

yes